bot/services.py

```diff
@@ -0,0 +1,61 @@
+from typing import Optional
+
+from aiohttp import ClientConnectorError
+
+from bot.bot import bot
+from bot.log import get_logger
+
+log = get_logger(__name__)
+
+FAILED_REQUEST_ATTEMPTS = 3
+PASTE_URL = "https://paste.pythondiscord.com/"
```

Comment:
The two places that use this URL duplicate the trailing slash here, resulting in a link in Discord such as `https://paste.pythondiscord.com//{code}?noredirect`